### DISCUSSION TITLE
Adding information to a table from Supabase Sign-in

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#6644] [Questions] [answered]
So this is the situation: My users sign in with a magic email link, and then are redirected to a basic "About You" page. I want to make it so users can then connect, say, their Facebook, and it add information from Facebook to that About You page, as well as a table.
Problem is, this wouldn't work using Supabase's normal auth system (afaik) because the user is already logged in.
How would I go about doing this, preferably using Supabase?

**Comment by GaryAustin1** [comment, marked as answer]
Technically you can signin with 3rd party auth after the magic link login and you have what ever 3rd party info that allows, but then the user can also login using the 3rd parties if you setup your login screen to deal with it somehow.
https://github.com/supabase/supabase/discussions/5827

You could also use the new edge functions to interact with your preferred 3rd party apis to get user info and store it, but I have nothing to point you to on doing that, although I'm sure there are articles on the web for doing that in a typescript function.
